### PR TITLE
Added snippets (Basic, Intermediate) See Desc.

### PR DESCRIPTION
**Pull request by PseudoCodeNerd**
####  Basic Level
is_divisible.md : Checks if the first numeric argument is divisible by the second one. Source: https://github.com/30-seconds/30-seconds-of-python/blob/master/snippets/is_divisible.md

#### Intermediate
chunk.md : Chunks a list into smaller lists of a specified size. Source 👍  https://github.com/30-seconds/30-seconds-of-python/blob/master/snippets/chunk.md

kth_smallest : Snippet to find the k-th smallest element in an array, given that k is less than size of the array.

#### Advanced Level
Will add tomorrow.

### CODE REVIEW

**Comment by PseudoCodeNerd** [comment]
Dang!
I thought so. Will remove it right now.

**Comment by PseudoCodeNerd** [comment]
So, @logankilpatrick should I let flatten.md stay ?

**Comment by logankilpatrick** [comment]
Yeah, I am no opposed to it.

**Comment by Chalarangelo** [comment]
> It's okay to do something that the core language does in a different way. Otherwise, it looks good.

Not really, we are trying to add code that complements native code and helps make working with the language easier, not reimplementing features. Exceptions to this are functions that have a better/more discernible name that the language and only if the alias is commonplace in another languge.

**Comment by PseudoCodeNerd** [comment]
Will remove  `flatten` in the next commit when I add a advanced level snippet 😄 

No specific reason for using `mod()`. Can replace it with `%` too.

File: snippets/chunk.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: chunk
+tags: list,intermediate
```
Comment (logankilpatrick) [inline snippets/chunk.md]:
```suggestion
tags: array,intermediate
```

Since lists don't exist in Julia.

File: snippets/chunk.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+Chunks a list into smaller lists of a specified size.
```
Comment (logankilpatrick) [inline snippets/chunk.md]:
```suggestion
Chunks an array into smaller arrrays of a specified size.
```

**Comment by PseudoCodeNerd** [comment]
@logankilpatrick Since I added a beginner snippet too, can you accept my task submission (https://codein.withgoogle.com/dashboard/task-instances/5245598962286592/)

Thanks.

**Pull request merged**